frontend/src/components/interviewCard.js

```diff
@@ -17,29 +17,57 @@ type Props = {
 class InterviewCard extends Component {
   constructor(props) {
     super(props)
+    this.state = {
+      modal: false
+    }
+    this.toggle = this.toggle.bind(this)
+    this.handleDeleteClick = this.handleDeleteClick.bind(this)
   }
 
   handleViewDetails() {
     this.props.onViewDetails(this.props.interview)
   }
 
+  async handleDeleteClick() {
+    await this.props.handleDelete()
+    this.toggle()
+  }
+
+  toggle() {
+    this.setState({
+      modal: !this.state.modal
+    })
+  }
+
   render() {
     return (
-      <Card className="candidate-card">
-        <CardBody>
-          <CardTitle>Overall Interview Score: {this.props.overallScore}</CardTitle>
-          <p>
-            <b>Round: </b> {this.props.interview.round}
-          </p>
-          <p>
-            <b>Interviewer: </b>
-            {this.props.interviewer}
-          </p>
-          <div>
-            <Button onClick={() => this.handleViewDetails()}>View Details</Button>
-          </div>
-        </CardBody>
-      </Card>
+      <>
+        <Modal isOpen={this.state.modal} toggle={this.toggle}>
+          <ModalHeader>
+            Are you sure you want to delete this interview? You won't be able to get it back.
+          </ModalHeader>
+          <ModalFooter>
+            <Button onCLick={this.toggle}>Return</Button>
```

Comment:
@rgychiu css pls
![image](https://user-images.githubusercontent.com/32147742/60155932-30c09600-97a0-11e9-80f0-73239181908b.png)

Reply:
Perhaps conditionally rendering the delete button based on who is viewing the component might be better. Not sure the feasibility of this though

Reply:
`margin-right`

Reply:
with the current pre-auth implementation I dont think its possible

Reply:
but once thats done it should be

Reply:
@tko22 still looks the same